src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+
+        String greeting="____________________________________________________________\n"
+                +" Hello! I'm Duke\n"
+                +" What can I do for you?\n"
+                + "____________________________________________________________\n";
+
+        System.out.println(greeting);
+        Scanner in= new Scanner(System.in);
+        String[] list= new String[100];
+        Task[] tasks = new Task[100];
+
+        //if mark or unmarked
+        boolean skip;
+        int indexing=1;
+        int i=0;
+
+        while(true)
+        {
+            skip=false;
+            String line=in.nextLine();
+            //if bye
+            if(line.equalsIgnoreCase("bye"))
+            {
+                break;
+            }
+            //getting index and line
+            list[i]=line;
+            String[] find_index = line.split(" ");
+
+            //Task description
+            String desc;
+
+            desc=line;
+            Task t = new Task(desc);
+
+
+            if(line.toLowerCase().contains("unmark"))
+            {
+                indexing=Integer.parseInt(find_index[1]);
+                tasks[indexing-1].markAsUnDone();
+
+
+
+                skip=true;
+                System.out.println("____________________________________________________________");
+                System.out.println("OK, I've marked this task as not done yet:\n"+
+                        "["+tasks[indexing-1].getStatusIcon()+"] "+ list[indexing-1]);
+                System.out.println("____________________________________________________________\n");
+            }
+
+
+            else if(line.toLowerCase().contains("mark"))
+            {
+                indexing=Integer.parseInt(find_index[1]);
+                tasks[indexing-1].markAsDone();
+
+
+
+                skip=true;
+                System.out.println("____________________________________________________________");
+                System.out.println("Nice! I've marked this task as done:\n"+
+                       "["+tasks[indexing-1].getStatusIcon()+"] "+ list[indexing-1]);
+                System.out.println("____________________________________________________________\n");
+
+            }
+
+            else if(line.toLowerCase().contains("todo") || line.toLowerCase().contains("deadline") || line.toLowerCase().contains("event") )
```

Comment:
This line is rather long and may be difficult to read and debug. Maybe by calling a method to check for this boolean statement, it will help to shorten and simplify the code, making it more readable.